lib/mysqlService.js

```diff
@@ -16,6 +16,10 @@ class MysqlService {
      * @param {Logger} [logger] - instance of hapiest-logger
      */
     constructor(writeConnectionConfig, readConnectionConfig, logger) {
+
+        this._writeConnectionConfig = writeConnectionConfig;
```

Comment:
Do we need the full configs saved?  If not, let's just save what we need, which appears to be the timezone setting.